python/code_challenges/hashtable_repeated_word.py

```diff
@@ -6,19 +6,14 @@ def first_repeated_word(string):
     regex = re.compile('[^a-zA-Z ]')
     stripped_string = regex.sub('', string)
 
-
-    # stripped_string = str(re.findall(r"\w+", string))
-
     string1 = stripped_string.lower().split()# convert string to lower and split
```

Comment:
Functionality is great here. Good use of method chaining.

Possible opportunity to change the variable name of `string1` to something that is more representative of what it is holding, both semantically and in terms of the type of data it holds. The split method creates a list of words, so maybe `word_list`? Something to consider when thinking of how to increase code readability.